WolvenKit.App/ViewModels/Editor/Tools/ImportExportViewModel.cs

```diff
@@ -506,13 +551,20 @@ private async Task ExecuteProcessAll()
                 }
                 await ImportWavs(wavs);
             }
-            else
+            if (IsExportsSelected)
             {
                 foreach (var item in ExportableItems)
                 {
                     await ExportSingle(item);
                 }
             }
+            if (IsConvertsSelected)
+            {
+                foreach (var item in ConvertableItems)
+                {
+                    await ConvertSingle(item);
```

Comment:
Call like this to avoid blocking: await Task.Run(() => ConvertSingleAsync(item));